ts/omm_de.ts

```diff
@@ -1398,7 +1398,7 @@ Soll die Selektion trotzdem entfernt werden?</translation>
     </message>
     <message>
         <source>The name of the object to render. Keep clear to render the whole scene. The name does not need to be unique. In this case, all matching objects are rendered. Perl-like regular expressions are supported.</source>
-        <translation>Der Name des zu rendernden Objektes. Leer lassen um die ganze Szene zu rendern. Der Pfad muss nicht unbedingt eindeutig sein. In diesem Fall werden alle passenden Objekte gerendert. Perl-artige reguläre Ausdrücke werden unterstüzt.</translation>
+        <translation>Der Name des zu rendernden Objektes. Leer lassen um die ganze Szene zu rendern. Der Name muss nicht unbedingt eindeutig sein. In diesem Fall werden alle passenden Objekte gerendert. Perl-artige reguläre Ausdrücke werden unterstüzt.</translation>
```

Comment:
Sure thing, if it's a path then it's a path. :)

I changed this because there is another, almost identical translation/source text like this where only `name` is substituted by `path`, so I figured they need to be translated differently to indictate some difference. 

For me the deeper question is maybe more the following though: Below are the two source texts related to this, coming from two different command line arguments specified in `subcommandlineparser.cpp`, but seeing these for the first time, I don't really understand how they differ, they seem to be the same (if like you say above, `name` is also a path type argument)? 

https://github.com/pasbi/ommpfritt/blob/8c90ef97a86720db6eaa698d4831ed76a579f949/src/subcommandlineparser.cpp#L41-L50

https://github.com/pasbi/ommpfritt/blob/8c90ef97a86720db6eaa698d4831ed76a579f949/src/subcommandlineparser.cpp#L57-L66

Reply:
You can see the difference here:
https://github.com/pasbi/ommpfritt/blob/8c90ef97a86720db6eaa698d4831ed76a579f949/src/maincli.cpp#L108

Note that it's forbidden to set both options at the same time, at least of them must not be set.
If the `object`-option is set, only the object name is matched, if the `path`-option is set, the object path is matched.

I find this makes sense, but I agree that the documentation could be more explicit on that.
One could argue that one could match always against the object path, and use the regex `'^_root_/.*/ObjectName$'` to match arbitrary paths.
From that perspective, the `object` option is just a convenient shortcut that is currently not used and might be removed if it creates too much confusion.
The `path` option, however is required to render the icons (see [generate-icons.py](https://github.com/pasbi/ommpfritt/blob/8c90ef97a86720db6eaa698d4831ed76a579f949/build-scripts/generate-icons.py)).

Reply:
Thanks for clarifying, if that's how it's intended the documentation is actually sufficient and clear - but the name argument then really *is* a name argument, not a path (that's what confused me - it's maybe a "name pattern", a "name expression" or a "name wildcard" if so supplied, but as I understand it now with your explanation it's definitely never a path for the `--object` option - please elaborate further if I still got this wrong somehow now :)).

Reply:
maybe it suffices if we

 1. change the "valueName" (the third argument, currently  `QObject::tr("NAME")` to `PATTERN` for both options
 2. make the option names more consistent: `object` -> `object-name` and `path` -> `object-path`.

Reply:
I added another commit to differentiate and (hopefully correctly :)) clarify the argument placeholders in the commandline documentation.

Reply:
Ah whoops sorry missed out on your post inbetween ... 1. I did more or less, of 2. I'm actually a bit wary because it will require further changes in more places to not break the icon generation e.g. for automated builds?

Reply:
regarding 1.: I'd prefer something with PATTERN, not REGEX, I think that's more standard (and technically, the regex is compiled by the application, the user only sets its pattern).

regarding 2.: no it won't because the icon generation uses the short command names (`-p`).
But updating the icon generation wouldn't be a big deal, if necessary.
Don't worry too much, if you break something, the automatic tests should fail and tell you what's wrong :wink: